src/utils/formatters.ts

```diff
@@ -0,0 +1,20 @@
+export const truncateChannel = (channel: string, pad: number = 9) => {
+  const regex = /^[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{34}$/;
```

Comment:
You could simplify this by using ranges instead of explicitly stating each character, for example `[0-9A-Za-z]` with a length specification for it.

Reply:
Resolved in https://github.com/ArkEcosystem/poc-ark-messenger/pull/5